core/regions/src/test/java/software/amazon/awssdk/regions/internal/util/EC2MetadataUtilsServer.java

```diff
@@ -213,7 +213,8 @@ protected String constructInstanceInfo() {
                + "\"version\":\"2010-08-31\","
                + "\"availabilityZone\":\"us-east-1b\","
                + "\"privateIp\":\"10.201.215.38\","
-               + "\"devpayProductCodes\":[\"bar\"]"
+               + "\"devpayProductCodes\":[\"bar\"],"
+               + "\"marketplaceProductCodes\":null"
```

Comment:
Can we make it  a non-null value and assert that value instead?

Reply:
Sure